### PR TITLE
esptool: add page

### PR DESCRIPTION
**Pull request by niwla23**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v4.2.1

### CODE REVIEW

File: pages/common/esptool.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# esptool
+
+> Espressif SoC serial bootloader utility.
+> More information: <https://docs.espressif.com/projects/esptool/en/latest/esp32/>.
+
+- Flash a firmware file to an ESP chip with given port and baud rate:
+
+`sudo esptool.py --port {{port}} --baud {{baud_rate}} write_flash 0x0 binary_esp8266.bin`
```
Comment (kbdharun) [inline pages/common/esptool.md:8]: Hi, Thanks for your contribution. I wasn't able to find the `--baud` flag in the documentation. What does it do?

Comment (niwla23) [reply pages/common/esptool.md:8]: from esptool.py --help:

  --baud BAUD, -b BAUD  Serial port baud rate used when flashing/reading

File: pages/common/esptool.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# esptool
+
+> Espressif SoC serial bootloader utility.
+> More information: <https://docs.espressif.com/projects/esptool/en/latest/esp32/>.
+
+- Flash a firmware file to an ESP chip with given port and baud rate:
```
Comment (kbdharun) [inline pages/common/esptool.md:6]:
```suggestion
- Flash a firmware file to an ESP chip with a given port and baud rate:
```

**Comment by niwla23** [comment]
@kbdharun can you review again?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/esptool.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# esptool
+
+> Espressif SoC serial bootloader utility.
+> More information: <https://docs.espressif.com/projects/esptool/en/latest/esp32/>.
+
+- Flash a firmware file to an ESP chip with a given port and baud rate:
+
+`sudo esptool.py --port {{port}} --baud {{baud_rate}} write_flash 0x0 binary_esp8266.bin`
```
Comment (navarroaxel) [inline pages/common/esptool.md:8]:
```suggestion
`sudo esptool.py --port {{port}} --baud {{baud_rate}} write_flash 0x0 {{path/to/firmware.bin}}`
```

File: pages/common/esptool.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# esptool
+
+> Espressif SoC serial bootloader utility.
```
[the remark this answers is not in the document]
Comment (niwla23) [reply pages/common/esptool.md:3]: I don't think that is a good description. Esptool is only made for espressif SoCs, not for all so i think that should be in there.

What about:

> Bootloader utility for Espressif chips (e.g. ESP8266)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**